cmplibrary/src/main/java/com/sourcepoint/cmplibrary/SpConsentLibImpl.kt

```diff
@@ -303,7 +303,8 @@ internal class SpConsentLibImpl(
             legIntCategories = legIntCategories.toList(),
             success = {
                 it?.let { spc ->
```

Comment:
what does `spc` stands for? the naming is confusing a bit

Reply:
@bohdan-go-wombat `spc` stands for `SPConsents`